packages/requests/src/lib/requests-result.spec.ts

```diff
@@ -434,6 +438,174 @@ describe('requests result', () => {
     `);
   });
 
+  it('should cache response data', () => {
+    jest.useFakeTimers().setSystemTime(new Date('2020-01-01'));
+
+    const { state, config } = createState(withEntities<Todo>());
+
+    const store = new Store({ state, config, name: 'todos' });
```

Comment:
We don't need all of these cases in this test, as we test only the cache property functionality. Keep it slim, please.

Reply:
Removed the addEntities instruction